docs/intro/tutorial03.txt

```diff
@@ -176,17 +176,17 @@ backend whose :setting:`APP_DIRS <TEMPLATES-APP_DIRS>` option is set to
 ``True``. By convention ``DjangoTemplates`` looks for a "templates"
 subdirectory in each of the :setting:`INSTALLED_APPS`.
 
-Within the ``templates`` directory you have just created, create another
-directory called ``polls``, and within that create a file called
-``index.html``. In other words, your template should be at
-``polls/templates/polls/index.html``. Because of how the ``app_directories``
+Within the :file:`templates` directory you have just created, create another
+directory called :file:`polls`, and within that create a file called
+:file:`index.html`. In other words, your template should be at
+:file:`polls/templates/polls/index.html`. Because of how the ``app_directories``
 template loader works as described above, you can refer to this template within
 Django simply as ``polls/index.html``.
```

Comment:
Please note that I didn't add `:file:` role to `polls/index.html` in this case, because it was used as a template name in the context of Django template loading argument, but I changed it in the lines prior to this one which was meant for files. I did not add `:file:` role to the template and static arguments of functions or template tags in the whole intro (tutorials).